### PR TITLE
Feature/crud cuisine categories

### PR DESCRIPTION
**Pull request by dgaleano201611**
## Summary
- Crud model cuisineCategories.

### CODE REVIEW

File: src/api/dishes/dishes.service.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+export const updateDishById = (id: string, input: any) => {
+  return prisma.dishes.update({
+    where: {
+      id_dish: id,
+    },
+    data: {
+      title: input.title,
+      description: input.description,
+      price: input.price, //ó parceInt
```
Comment (ht1204) [inline src/api/dishes/dishes.service.ts:39]: is necessary a parseInt or not?

File: src/api/dishes/dishes.service.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      restaurantsId_restaurant: input.restaurantsId_restaurant,
+      dishes_categoriesId_dishes_category:
+        input.dishes_categoriesId_dishes_category,
+      order_details: input.order_details,
```
Comment (ht1204) [inline src/api/dishes/dishes.service.ts:43]: Destructuring would be better yet.

**Review by ht1204 — CHANGES_REQUESTED** [review]
You have done a great job, but please, address the comments. More descriptive PRs would be better yet for a better understanding, I'm attentive to any updates.

File: src/api/dishes/dishes.service.ts (hunk 1 above)
Comment (dgaleano201611) [reply src/api/dishes/dishes.service.ts]: parseInt is'n necessary because input.price is a number

File: src/api/dishes/dishes.service.ts (hunk 2 above)
Comment (dgaleano201611) [reply src/api/dishes/dishes.service.ts]: the destructuring in updateDishById methd was made

**Review by ht1204 — APPROVED** [review]

File: src/api/dishes/dishes.service.ts (hunk 1 above)
Comment (ht1204) [reply src/api/dishes/dishes.service.ts]: ok, thx 4 clarify.

**Pull request merged**